_pytest/fixtures.py

```diff
@@ -1130,7 +1130,40 @@ def getfixturedefs(self, argname, nodeid):
         else:
             return tuple(self._matchfactories(fixturedefs, nodeid))
 
+    @classmethod
+    def _splitnode(cls, nodeid):
```

Comment:
It appears that this can be a `staticmethod` instead. The class isn't referenced anywhere in this method.

Reply:
I'm not a fan of staticmethods, though mainly for inheritance related reasons and doubt anyone is subclassing fixtruemanager, let alone doing so and then wanting to change these. It feels like they should probably be plain functions in a 'node' module somewhere but hopefully someone more familiar with pytest can guide me on that... (see also my comment about `_splitnode` in the description.

Reply:
since both helpers are completely unrelated to the class, how about just moving them out of the class instead